File: 58-LengthOfLastWord/58-LengthOfLastWord.js
// Last updated: 10/03/2026, 10:03:49
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let count = 0;

    for (let i = s.length - 1; i >= 0; i--) {

        if (s[i] === " " && count === 0) {
            continue;
        }

        if (s[i] === " ") {
            break;
        }

        count++;
    }

    return count;
};

File: 58-LengthOfLastWord/58-LengthOfLastWord.cpp
// Last updated: 13/05/2026, 13:11:02
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count=0;
        for(int i=s.size()-1;i>=0;i--){
            if(s[i]==' ' && count==0){
                continue;
            }
            if(s[i]==' '){
                break;
            }
            count++;
        }
    return count;
    }
};